pkg/tagger/types/go.mod

```diff
@@ -1,3 +1,7 @@
 module github.com/DataDog/datadog-agent/pkg/tagger/types
 
 go 1.22.0
+
+replace github.com/DataDog/datadog-agent/comp/core/tagger/origindetection => ../../../comp/core/tagger/origindetection
+
+require github.com/DataDog/datadog-agent/comp/core/tagger/origindetection v0.62.0-rc.1
```

Comment:
does this package has a limited set of dependencies? else I suppose it is a risk to increase the RSS and binary size

Reply:
Yes it is small: https://github.com/DataDog/datadog-agent/blob/main/comp/core/tagger/origindetection/go.mod